modules/sbo-binding-workloads-that-are-not-compliant-with-PodSpec.adoc

```diff
@@ -6,14 +6,16 @@
 [id="sbo-binding-workloads-that-are-not-compliant-with-PodSpec_{context}"]
 = Binding secondary workloads that are not compliant with PodSpec
 
-[role="_abstract"]
 A typical scenario in service binding involves configuring the backing service, the workload (Deployment), and {servicebinding-title}. Consider a scenario that involves a secondary workload (which can also be an application Operator) that is not compliant with PodSpec and is between the primary workload (Deployment) and {servicebinding-title}.
 
 For such secondary workload resources, the location of the container path is arbitrary. For service binding, if the secondary workload in a CR is not compliant with the PodSpec, you must specify the location of the container path. Doing so projects the binding data into the container path specified in the secondary workload of the `ServiceBinding` custom resource (CR), for example, when you do not want the binding data inside a pod.
 
-{servicebinding-title} provides an option to configure the value of where the container path or secret path is and bind these paths at a custom location. This option is available only for the `binding.operators.coreos.com` API group when the binding data is projected as environment variables.
+{servicebinding-title} provides an option to configure the value of where the container path or secret path is and bind these paths at a custom location.
```

Comment:
- Does this mean you can configure **where** the path information is (the location of the path information), or that you can configure the **path** itself? 
- Does this mean you can bind these paths *at* a custom location (control where the binding happens), or that you can bind these paths **to** a custom location?

Can we update the wording to answer these questions and rephrase the information in terms of the user's needs? 

What about:
```suggestion
In {servicebinding-title}, you can configure the container path or secret path, and bind these paths to a custom location.
```
OR 
```suggestion
In {servicebinding-title}, you can configure the location of the container path or secret path, and bind these paths at a custom location.
```

Reply:
Preferred version:
In {servicebinding-title}, you can configure the path of where containers or secrets reside within a workload and bind these paths at a custom location.

Reply:
Fixed